starter/README.md

```diff
@@ -3,21 +3,16 @@
 Servlet initial parameters may be passed via `@Component` annotation parameter `property`, e.g.
 
 ```java
-@Component(immediate = false, service = Servlet.class, property = {
-  HttpWhiteboardConstants.HTTP_WHITEBOARD_SERVLET_INIT_PARAM_PREFIX +
-  InitParameters.SERVLET_PARAMETER_COMPATIBILITY_MODE + "=false"}) 
+@Component(immediate = false, service = Servlet.class, property = { "some.property = someValue" })
 ```
 or via some annotation like `VaadinMode` which is available in the example.
 
-Same way can be used to set production mode via the initial parameters.
-Another way to set these values is explicitly set them via the vaadin maven plugin configuration:
+Another way to set the property values is explicitly set them via the vaadin maven plugin configuration:
 
 ```xml
- <configuration>
-    <compatibilityMode>false</compatibilityMode>
-    <productionMode>false</productionMode>
+<configuration>
```

Comment:
This README is not about Vaadin and Vaadin maven plugin.
It describes how to configure Vaadin Servlet in OSGi.
As a result the change has no any sense.

This is what it said previously:
- you may configure compat mode (which is absent now) or production mode either via Servlet init parameter (above)
- or via maven plugin configuration .

This is just a clarification of Vaadin documentation ( which I hope we have) that you may use init parameter to set the DeploymentConfiguration property or you may specify _some_ properties in the token file via maven configuration.

In the master version it means: the first case configures the `DeploymentConfiguration` per `Servlet` , the second case configures `ApplicationConfiguration` via token file.

Reply:
Thanks! Brought back the production mode parameter here.